gravitee-am-identityprovider/gravitee-am-identityprovider-common/src/main/java/io/gravitee/am/identityprovider/common/oauth2/authentication/AbstractOpenIDConnectAuthenticationProvider.java

```diff
@@ -311,6 +308,8 @@ protected void generateJWTProcessor(OpenIDConnectIdentityProviderConfiguration c
                         keyProcessor = new HMACKeyProcessor();
                         keyProcessor.setJwkSourceResolver(new MACJWKSourceResolver(resolverParameter));
                         break;
+                    default:
```

Comment:
break in default?

Can we not use a new switch here?

Reply:
https://sonarcloud.io/organizations/gravitee-io/rules?open=java%3AS131&rule_key=java%3AS131